<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projekt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Canvas" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Translacja</string>
    </property>
    <widget class="QSlider" name="translationX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="translationY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="translationZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotacja</string>
    </property>
    <widget class="QSlider" name="rotationX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotationY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="rotationZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Skalowanie</string>
    </property>
    <widget class="QSlider" name="scalingX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scalingY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="scalingZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
